test/integration/mon/serverspec/mon_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'serverspec'
+
+set :backend, :exec
+
+load_thres = if %w(ppc64 ppc64le).include?(os[:arch])
+               '-w 14,9,4 -c 18,13,8'
```

Comment:
Ah, I guess this is for the test kitchen, where cpu count would probably be lower.